### DISCUSSION TITLE
Increase Icon Size / Text Size

### DISCUSSION THREAD

**Discussion by breitgo** [Blackymas/NSPanel_HA_Blueprint#2201] [Q&A]
- Currently doing my first steps with NSPanel and wonder if there is a way to modify it a bit for visually impaired. E.g. I would like to increase the icon size which gets me from the home page to TRV controls. 32px is a bit small here. 

- Same for text for hardware button 1&2. Here also the question if I can modify "push & hold" to get me to e.g. the climate page. 

- How to remove / replace the background image?

**Comment by wojciechczyz** [comment]
I second your opinion. Making variable icon sizes would help. Panel is small and seeing/clicking is not easy at times.

**Reply by pana800** [reply]
It would be great if we can have the option of changing to bigger size (i.e less number of buttons) on the home screen. Thanks
